### DISCUSSION TITLE
Is it possible to use some kind of placeholder as a level name?

### DISCUSSION THREAD

**Discussion by nmills3** [itzg/docker-minecraft-server#2473] [Q&A] [answered]
So basically i want to make it so that my level name is based off of the current snapshot version that the server is running. Is there any way to set the level name as something like "world-%VERSION%" or something similar?

My goal is to have the server create a new world for each snapshot version that gets release. that way i can just restart the container and then the new snapshot server will get auto downloaded and a new world will be created. But if i want i could still go back and access the old worlds

**Comment by itzg** [comment, marked as answer]
No, but that's a great idea! I'm going to convert this into an enhancement issue.

**Reply by itzg** [reply]
Created https://github.com/itzg/docker-minecraft-server/issues/2474
